pymc3/plots/pairplot.py

```diff
@@ -0,0 +1,162 @@
+import warnings
+import matplotlib.gridspec as gridspec
```

Comment:
This line should be inside the try-except block below. Additionally you can change it to `from matplotlib import gridspec`